looprpc/server.proto

```diff
@@ -19,6 +19,20 @@ service SwapServer {
         returns (ServerLoopInQuoteResponse);
 }
 
+/**
+This enum defines the protocol versions that clients may adhere to. Note that
+this is not a flagged enum. If a particular protocol version adds a feature,
+then in general all the preceding features are also supported. Exception to this
+is when features get deprecated.
+*/
+enum ProtocolVersion {
+    /// No protocol version reported at all.
+    LEGACY = 0;
+
+    /// Client reports its supported protocol version.
+    REPORT_VERSION = 1;
```

Comment:
So this enum will be extended to have additional protocol versions? If that's the case the name `REPORT_VERSION` and its comment don't make sense to me - makes me think we're going to provide a specific version value somewhere.

Also, how are these protocol versions tracked? By adding an enum to this proto?
Thinking about the case of a non-loop client (if we're accounting for that), how do I know what version I'm supporting?

Reply:
I can agree with @carlaKC here, the naming is a bit confusing. Perhaps instead of calling it `ProtocolVersion` we could call itt `SwapProtocol` (if I understand the purpose correctly), and have better naming for the individual protocol identifiers.

Reply:
I think that the confusion is that `REPORT_VERSION` itself is a feature that a client can support? 

The idea is indeed that with every protocol change we do on the server, we add a value to the enum. Then the next client release will start sending that enum value.

Reply:
Loop is all about swaps, so `SwapProtocol` doesn't add more information than just `Protocol` in my opinion.

Suggestions for what to rename `REPORT_VERSION` to?

Reply:
I think a better question is how will we use `REPORT_VERSION`? Isn't `LEGACY` the only version we use now?

Reply:
Can also call it `MULTI_LOOP_OUT`?

Reply:
That is much better :) I'd be happy with that.

Reply:
Updated